assets/sass/05-blocks/file/_editor.scss

```diff
@@ -1,5 +1,11 @@
 .wp-block-file {
 
+	.wp-block-file__textlink {
```

Comment:
The link style need to be a solid 1 pixel underline to match the themes other link styles.

The :hover style needs to be an underline, but dotted instead of solid.

Reply:
Unless there is a reason for using a border instead of the text decoration in the editor?